oauth2_provider/models.py

```diff
@@ -132,6 +132,10 @@ class AbstractApplication(models.Model):
     created = models.DateTimeField(auto_now_add=True)
     updated = models.DateTimeField(auto_now=True)
     algorithm = models.CharField(max_length=5, choices=ALGORITHM_TYPES, default=NO_ALGORITHM, blank=True)
+    allowed_origins = models.TextField(
+        blank=True,
+        help_text=_("Allowed origins list to enable CORS, space separated"),
+    )
```

Comment:
Don't we need to add a default value ?
I recently had to add one to another field in #1319

Reply:
@tonial definitely! Missed that migration will fail if row exist in `application` table.
Thank you!

Reply:
@tonial is this resolved to your satisfaction?